blocks/count-cards/count-cards.js

```diff
@@ -0,0 +1,48 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  /* change to ul, li */
+  const ul = document.createElement('ul');
+  [...block.children].forEach((row) => {
+    const li = document.createElement('li');
+    while (row.firstElementChild) li.append(row.firstElementChild);
+    [...li.children].forEach((div) => {
+      if (div.children.length === 1 && div.querySelector('img')) {
+        div.classList.add(...['count-card-image']);
+        div.getElementsByTagName('img')[0].classList.add(...['icon-svg-ethos', 'lazyloaded']);
+      } else if (div.children.length === 3) {
+        div.querySelectorAll('h4')
+          .forEach((e) => {
+            e.classList.add(...['ethos-2-theme', 'heading-5', 'count-card-title']);
+          });
+        div.querySelectorAll('p')
+          .forEach((e) => {
+            e.classList.add(...['ethos-2-theme', 'body-2', 'count-card-text']);
+          });
+        const aTag = div.getElementsByTagName('a')[0];
+        aTag.classList.add(...['link', 'qui-link', 'ng-star-inserted', 'standalone']);
+        aTag.setAttribute('cta-appearance', 'link');
+        aTag.setAttribute('target', '_self');
+        aTag.setAttribute('mat-button', 'false');
+        aTag.setAttribute('disabled', 'false');
+        const pTag = aTag.parentNode;
+        pTag.classList.remove(...['count-card-text']);
+        const wcCtaTag = document.createElement('qui-wc-cta');
+        wcCtaTag.setAttribute('cta-type', 'link');
+        wcCtaTag.setAttribute('appearance', 'standalone');
+        wcCtaTag.setAttribute('href', aTag.getAttribute('href'));
+        wcCtaTag.setAttribute('target', '_self');
+        wcCtaTag.setAttribute('icon-position', 'right');
+        wcCtaTag.className = 'qui-cta';
+        wcCtaTag.appendChild(aTag);
+        pTag.appendChild(wcCtaTag);
```

Comment:
The code from main should now be handling  the link decoration. You can merge main back in your branch and remove this code.

Reply:
fetched latest changes and removed this section of code.